### ISSUE TITLE
dG y'=y does not work

### ISSUE THREAD

**Issue by nrfulton** [LS-Lab/KeYmaeraX-release#38]

**Comment by nrfulton** [comment]
This bug is now fixed, but there is a larger class of bugs that we could categorize as "ghosting doesn't work even though I should be allowed to ghost this ODE".

These bugs are caused by missing cases in [`DifferentialHelper.parseGhost`](https://github.com/LS-Lab/KeYmaeraX-release/blob/77d94cfa5710fef53ec460c5fb4c5628475db3dd/keymaerax-core/src/main/scala/edu/cmu/cs/ls/keymaerax/btactics/helpers/DifferentialHelper.scala#L99).

I tried to make sure most other possible forms are supported as well, but related bugs might crop up.

Pragmatic user advice until we iron out all of the missing cases: just use the full form `y' = a*y + b`. So e.g., choose `a` or `b` as 0 if you're not using them.
